templates/zerver/api/changelog.md

```diff
@@ -20,6 +20,12 @@ format used by the Zulip server that they are interacting with.
 
 ## Changes in Zulip 5.0
 
+
+**Feature level 115**
+
+* Mobile push notifications about stream messages now include the
+  `stream_id` field.
```

Comment:
Bumping feature level here is correct, right? And am I missing any additional documentation that needs to be updated?

Reply:
Yes, that's correct.  We don't have another place we document the push notifications format, so I don't think there is another place to update at present.